src/electionguard/serialize.py

```diff
@@ -1,31 +1,39 @@
+from datetime import datetime
 from io import TextIOWrapper
 import json
 import os
 from pathlib import Path
 from typing import Any, List, Type, TypeVar, Union
 
-from pydantic import BaseModel, PrivateAttr
+from dacite import Config, from_dict
 from pydantic.json import pydantic_encoder
-from pydantic.tools import parse_raw_as, parse_obj_as, schema_json_of
-
-Private = PrivateAttr
-
-
-class Serializable(BaseModel):
-    """Serializable data object intended for exporting and importing"""
-
-    class Config:
-        """Model config to handle private properties"""
-
-        underscore_attrs_are_private = True
+from pydantic.tools import parse_raw_as, schema_json_of
 
+from .ballot_box import BallotBoxState
+from .manifest import ElectionType, ReportingUnitType, VoteVariationType
+from .group import ElementModP, ElementModQ
+from .proof import ProofUsage
 
 _T = TypeVar("_T")
 
 _indent = 2
 _encoding = "utf-8"
 _file_extension = "json"
 
+_config = Config(
```

Comment:
This tool takes a config that will cast all the elements correctly. This solves for the particular problem where BigIntegers are brought in by strings on pydantic due to the lack of overrides. I don't really like pulling in another tool but at least this is top down and not bottom up.

Reply:
Agreed and if it gives the desired format that is worth it.